server/server.go

```diff
@@ -159,7 +160,7 @@ func (srv *Server) startServer(_ context.Context) error {
 		MaxScanLimit: srv.cfg.MaxScanLimit, MinScanLimit: srv.cfg.MinScanLimit,
 	})
 
-	manager, err := cluster.NewManagerImpl(storage, srv.etcdCli, srv.etcdCli, srv.cfg.StorageRootPath, srv.cfg.IDAllocatorStep, srv.cfg.EnableSchedule)
+	manager, err := cluster.NewManagerImpl(storage, srv.etcdCli, srv.etcdCli, srv.cfg.StorageRootPath, srv.cfg.IDAllocatorStep, srv.cfg.EnableSchedule, scheduler.TopologyType(srv.cfg.TopologyType))
```

Comment:
Maybe we should add a method to parse the TopologyType from the raw string rather than just use it.